posthog/hogql_queries/insights/test/test_lifecycle_query_runner.py

```diff
@@ -36,6 +36,544 @@ def create_action(**kwargs):
     return action
 
 
+class TestLifecycleQueryRetentionGroupAggregation(ClickhouseTestMixin, APIBaseTest):
```

Comment:
Nit: We've usually been pretty lazy and adding all tests inside one large class. This can make it easier to "run all" in PyCharm, but we use that almost never, and I'm not against splitting it up. However might be good to move the new stuff to the end of the file.